site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -37,34 +36,21 @@ All of the FusionAuth Docker images may be found on https://hub.docker.com/u/fus
 
 The FusionAuth Docker Compose yaml files may be found on https://github.com/FusionAuth/fusionauth-containers[FusionAuth containers repository].
 
-If you're looking for a complete configuration to get up and running quickly, use our Docker Compose example. `docker-compose.yml` will install FusionAuth without the enhanced search capability that Elasticsearch provides.  If you would rather use FusionAuth with Elasticsearch for improved searching, make sure you include the `docker-compose.override.yml`.
+If you're looking for a complete configuration to get up and running quickly, use our Docker Compose example. `docker-compose.yml` will install FusionAuth with ElasticSearch, but you can link:/docs/v1/tech/tutorials/switch-search-engines/[switch between the different search engines].
 
 === Installing FusionAuth
 
 include::docs/v1/tech/installation-guide/_docker-compose-files.adoc[]
 
-=== Installing FusionAuth with Elasticsearch
-
-:with_elasticsearch:
-include::docs/v1/tech/installation-guide/_docker-compose-files.adoc[]
-:with_elasticsearch!:
-
 === The Docker Compose Files
 
-The following is the default `docker-compose.yml` file, which installs FusionAuth without the Elasticsearch engine.
+The following is the default `docker-compose.yml` file, which installs FusionAuth.
 
 [source,yaml,title=docker-compose.yml]
 ----
 include::https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/master/docker/fusionauth/docker-compose.yml[]
```

Comment:
I think that this is fine - but my yml linter does not like this definition:

![image](https://user-images.githubusercontent.com/16090626/188980668-b145a7ab-a46c-4a12-bf7f-648ef898a2bc.png)

Reply:
that's because the removed it from the v3 spec, but it still works, and there is not another good alternative as far as I am aware.